tensorflow/core/util/device_name_utils.cc

```diff
@@ -162,6 +162,16 @@ bool DeviceNameUtils::ParseFullName(StringPiece fullname, ParsedName* p) {
       }
       progress = true;
     }
+    if (str_util::ConsumePrefix(&fullname, "/sycl:") ||
```

Comment:
#4359 is very useful, thanks for the link! :)
Does it mean we need to copy and pasting all the existing TF tests and duplicated them with SYCL ones?

We do plan on supporting all the existing tests in the future.